src/components/JobLogs/JobLogs.js

```diff
@@ -42,13 +48,25 @@ export class JobLogs extends Component {
 
     this.visibleColumns = [
       'fileName',
+      'log',
       'jobProfileName',
       'jobExecutionHrId',
       'completedDate',
       'runBy',
     ];
   }
 
+  createLogButton = id => (
+    <Link to={{ pathname: `${this.props.match.path}/log/${id}` }} target="_blank" style={{ color: '#fff' }}>
```

Comment:
There is no need to use `Link`. Use `Button` `to` prop. Check out usages with *ui-data-import* application.